src/sdk/util/passphrase.ts

```diff
@@ -0,0 +1,39 @@
+import { topPasswords } from '../topPassphrases';
+import { randomBytes } from './crypto';
+
+export function generateRandomKeywords(): string[] {
+    const randomIndices: number[] = [];
+    const maxIndex = topPasswords.length;
+
+    while (randomIndices.length < 6) {
+        // Generate a random 4-byte number
+        const randomBytesArray = randomBytes(4);
```

Comment:
It would be cleaner if you created a new function `randomNumber(min: number, max: number)` in crypto.ts